src/makeCalculator.js

```diff
@@ -37,7 +37,31 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+    operate(operation, a) {
+      operation.call(this, a);
+
+      return this;
+    },
+    reset() {
+      this.result = 0;
+
+      return this;
+    },
+    add(a) {
```

Comment:
```suggestion
    },
    
    add(a) {
```

It would be better for readability to add empty line between two methods